packages/solo/src/access-token.js

```diff
@@ -32,7 +32,7 @@ export async function getAccessToken(port) {
   await fs.promises.mkdir(sharedStateDir, { mode: 0o700, recursive: true });
 
   // Ensure an access token exists.
-  const { kvStore, commit, close } = openSwingStore(sharedStateDir, 'state');
+  const { kvStore, commit, close } = openSwingStore(sharedStateDir);
```

Comment:
This looks to me like it's using the SwingStore API but its own database (i.e., not the swingset database), which may be overkill but aside from (modest) overhead seems as good a way to store anything given the prior that you are going to store it.  But Michael can tell use whether or not I'm wright about that.

Reply:
@FUDCo is correct.  This use of `swing-store-simple` had no relationship to SwingSet.

It was not overkill when the database was a simple jsonlines file.  I really don't want to make it LMDB because that introduces portability concerns for what should just be a text file.

Quoting myself:
>> Do you need more than one value? Maybe this could just go into a single one-line file.
> 
> I can easily implement what I need. Just please don't remove the jsonlines implementation before I have a chance to do that.

Here is a sample of the current contents of the file:

```js
soil:20210528 michael$ cat ~/.agoric/swingset-kernel-state.jsonlines 
["accessToken/8000","4S0..."]
["accessToken/7999","p0Q..."]
["accessToken/8001","Ifr..."]
["accessToken/9000","ArB..."]
["accessToken/6892","7P-..."]
```